### PR TITLE
fix glooctl kubectl timeouts

### PR DESCRIPTION
**Pull request by ashleywang1**
BOT NOTES: 
resolves https://github.com/solo-io/gloo/issues/2771

### CODE REVIEW

File: docs/content/operations/debugging_gloo/_index.md [hunk 1]
```diff
@@ -177,7 +177,7 @@ Each Gloo control plane component comes with a optional debug port that can be e
 kubectl port-forward  -n gloo-system deploy/gloo  9091:9091
 ```
 
-Now you can navigate to [http://localhost:9091](http://localhost:9091) and you get a simple page with some additional endpoints:
+Now you can navigate to http://localhost:9091 and you get a simple page with some additional endpoints:
```
Comment (kdorosh) [inline docs/content/operations/debugging_gloo/_index.md:180]: why this change? I slightly prefer the clickable links

File: projects/gloo/cli/pkg/cmd/check/proxy_prom_stats.go [hunk 2]
```diff
@@ -40,7 +40,8 @@ func checkProxyPromStats(ctx context.Context, glooNamespace string, deploymentNa
 	freePort, err := cliutil.GetFreePort()
 	if err != nil {
 		fmt.Println(errMessage)
-		return false, err
+		fmt.Println(err)
+		return true, nil
```
Comment (kdorosh) [inline projects/gloo/cli/pkg/cmd/check/proxy_prom_stats.go:44]: i still think we should return errors here

File: docs/content/operations/debugging_gloo/_index.md (hunk 1 above)
Comment (ashleywang1) [reply docs/content/operations/debugging_gloo/_index.md:180]: They still show up as clickable even without the extra formatting!
